fsButler/utils.py

```diff
@@ -722,13 +772,16 @@ def getParent(objId, fsButler, dataType='deepCoadd'):
     parent = src[objId == src.get("id")][0]
     return parent
 
+
 def getMultId(cat):
-   bands = getCatBands(cat)
-   multIds = np.zeros((len(cat),), dtype=[(b, 'int64') for b in bands])
-   for b in bands:
-       multIds[b] = cat.get('multId.'+b)
-   return multIds
+    bands = getCatBands(cat)
+    multIds = np.zeros((len(cat),), dtype=[(b, 'int64') for b in bands])
+    for b in bands:
+        multIds[b] = cat.get('multId.'+b)
+    return multIds
 
+
+''' Commented out since we don't need to perform the explaratory analysis analysis anymore.
```

Comment:
Typo (“explaratory analysis analysis”).

Prefer removing code to commenting it out. Obviously, that should be on a separate commit, and the commit message should explain what the analysis in question was.